pkgs/games/manaplus/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, SDL2, SDL2_image, SDL2_ttf, SDL2_mixer, SDL2_net,
+SDL2_gfx, zlib, physfs, curl, libxml2, libpng, pkgconfig, mesa, autoreconfHook
+}:
+stdenv.mkDerivation rec {
+  version = "1.9.3.23";
+  name = "manaplus-" + version;
+
+  src = fetchurl {
+    url = "https://download.evolonline.org/manaplus/download/${version}/manaplus-${version}.tar.xz";
+    sha256 = "1ky182p4svwdqm6cf7jbns85hidkhkhq4s17cs2p381f0klapfjz";
+  };
+
+  buildInputs = [
```

Comment:
`autoreconfHook` and `pkgconfig` belong under `nativeBuildInputs`.

Also I believe `pkg-config` is preferred to `pkgconfig` now.